src/Backend/Core/Js/jquery/jquery.backend.js

```diff
@@ -1500,7 +1500,7 @@
     e && e.preventDefault()
 
     var collection = $('#' + selector)
-    var list = collection.find('ul.js-collection')
+    var list = $(collection.find('ul.js-collection')[0])
```

Comment:
I think a nicer solution would be `collection.find('ul.js-collection').first()`, could you try that?

Reply:
I agree, changed it as you suggested.